src/main/java/seedu/todo/ui/InputHandler.java

```diff
@@ -30,10 +30,12 @@ public void processInput(String input) {
     }
     
     private Controller[] instantiateAllControllers() {
-        return new Controller[] { new AddController(),
-                                  new ListController(),
-                                  new DestroyController(),
-                                  new UpdateController() };
+        return new Controller[] { 
+                new AddController(),
+                new ListController(),
+                new DestroyController(),
+                new UpdateController() 
+                };
```

Comment:
Please revert this "fix", it's more unreadable than the original chunk, and doesn't follow coding standards.

Standard is to _align the new line with the beginning of the parent element_, and regardless, line continuation indentation should be 4 spaces, not 8.